app/views/state_file/questions/nj_county/edit.html.erb

```diff
@@ -1,5 +1,5 @@
 <%
-    title = t(".title", filing_year: current_tax_year)
+    title = t(".title_html", filing_year: current_tax_year)
```

Comment:
[pebble] using html as the page title messes up how it appears in the browser. There need to be separate page title and h1 title html keys for pages with html in the title.

<img width="356" alt="Screenshot 2025-01-10 at 3 36 33 PM" src="https://github.com/user-attachments/assets/cdcf0f52-bacc-4deb-9fd4-b20e6601f33f" />

Reply:
here's another way to go about that: https://github.com/codeforamerica/vita-min/pull/5330